millo.js

```diff
@@ -21,4 +21,134 @@ function listBoards() {
 
 };
 
-function createBoard(boardName) {};
\ No newline at end of file
+function createBoard(boardName) {
+    for (let check in hello) {
```

Comment:
we can also just use truthy indexing:
`if ( hello[boardName] ) { return "Board already exists" }`